src/test/client/TestClient.h

```diff
@@ -20,9 +20,27 @@
 #include "msg/Messenger.h"
 #include "mon/MonClient.h"
 #include "osdc/ObjectCacher.h"
-
+#include "client/MetaRequest.h"
 #include "client/Client.h"
 
+#define dout_subsys ceph_subsys_client
+
+class TestNewOpsClient : public Client {
```

Comment:
```suggestion
class ClientScaffold : public Client {
```

Reply:
Done, thanks for this, saved some time thinking for the name :)